apps/bare-expo/ios/Podfile

```diff
@@ -28,11 +28,6 @@ abstract_target 'BareExpoMain' do
 
   use_react_native!(path: config['reactNativePath'])
 
-  pod 'EXDevMenu', path: '../../../packages/expo-dev-menu', configurations: :debug
```

Comment:
This line was added to remove the `expo-dev-menu` from release builds and I think we should leave that.

Reply:
hmm we should have some heuristic for this functionality that can be statically generated for managed workflow support. Any ideas?

Reply:
We'll remove it after the next publish of the RN CLI -  https://github.com/react-native-community/cli/pull/1298